server/notification-providers/discord.js

```diff
@@ -14,14 +14,24 @@ class Discord extends NotificationProvider {
 
         try {
             const discordDisplayName = notification.discordUsername || "Uptime Kuma";
+            let threadQuery = "";
+
+            if (notification.discordChannelType === "postToThread") {
+                threadQuery = "?thread_id=" + notification.threadId;
```

Comment:
Append query string to an unknown URL is not completely safe. Should use parser.